laalaa/settings/base.py

```diff
@@ -111,6 +113,23 @@
     'EXCEPTION_HANDLER': 'advisers.views.custom_exception_handler'
 }
 
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
+        'LOCATION': root('cache'),
+    }
+}
+
+CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack']
+CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
+BROKER_URL = 'amqp://%s:%s@%s//' % (
+    os.environ.get('RABBITMQ_USER', 'guest'),
+    os.environ.get('RABBITMQ_PASS', 'guest'),
+    os.environ.get('HOST_IP', '127.0.0.1'),
+)
```

Comment:
Use `.format` in preference to the `%` operator:
```python
BROKER_URL = `amqp://{user}:{pass}@{host}//'.format(
    user=os.environ.get('RABBITMQ_USER', 'guest'),
    pass=os.environ.get('RABBITMQ_PASS', 'guest'),
    host=os.environ.get('HOST_IP', '127.0.0.1'))
```
or even (if you know the environment variables will always be present):
```python
BROKER_URL = 'amqp://{RABBITMQ_USER}:{RABBITMQ_PASS}@{HOST_IP}//'.format(**os.environ)
```